usr/share/tb-profile-i2p/profile.i2p/preferences/extension-overrides.js

```diff
@@ -0,0 +1,127 @@
+# Overrides for Extension Preferences
```

Comment:
What's the source of this file? Own creation? License?

Reply:
This was taken directly from the default profile in Tor Browser Bundle. I decided to leave it unaltered based on the prior forum discussion about whether I should remove the unnecessary defaults. Since they didn't have any effect by default my understanding was that I should just leave them alone.